pkg/controller/servicemesh/controlplane/pruner.go

```diff
@@ -42,8 +42,8 @@ var (
 		gvk("networking.k8s.io", "v1", "NetworkPolicy"),
 		gvk("rbac.authorization.k8s.io", "v1", "RoleBinding"),
 		gvk("rbac.authorization.k8s.io", "v1", "Role"),
-		gvk("admissionregistration.k8s.io", "v1beta1", "MutatingWebhookConfiguration"),
-		gvk("admissionregistration.k8s.io", "v1beta1", "ValidatingWebhookConfiguration"),
+		gvk("admissionregistration.k8s.io", "v1", "MutatingWebhookConfiguration"),
```

Comment:
should we remove beta resources from this list? or just add the v1 to it so both end up pruned?

Reply:
This is okay, so long as the clusters we're deploying to support v1.  The resource is the same, it's just whether you're looking at it through the eyes of v1 or v1beta1.  Just like smcp v1 vs v2.